matRad_calcParticleDoseBixel.m

```diff
@@ -63,22 +60,27 @@
     X(radDepths >= max(depths),1) = 0;
         
     % compute lateral sigmas
-    sigmaSq_Narr = X(:,2).^2 + sigmaRashi.^2 + SigmaIni^2;
-    sigmaSq_Bro  = X(:,4).^2 + sigmaRashi.^2 + SigmaIni^2;
+    sigmaSq_Narr = X(:,2).^2 + sigmaRashi.^2 + sigmaIni^2;
+    sigmaSq_Bro  = X(:,4).^2 + sigmaRashi.^2 + sigmaIni^2;
     
     % calculate lateral profile
     L_Narr =  exp( -radialDist_sq ./ (2*sigmaSq_Narr))./(2*pi*sigmaSq_Narr);
     L_Bro  =  exp( -radialDist_sq ./ (2*sigmaSq_Bro ))./(2*pi*sigmaSq_Bro );
     L = baseData.LatCutOff.CompFac * ((1-X(:,3)).*L_Narr + X(:,3).*L_Bro);
 
     dose = X(:,1).*L;
+    
+    % This is dangerous but we need this because, using a currIx evaluated
+    % once for every ray, we get a deeper radiation depths that with
+    % interpolation can give negative results.
+    dose(dose<0) = 0;
```

Comment:
Dose can be negative in cases where we use option 'cubic' in interpolation. 
If we want to remove this option,  we have to use linear interpolation only.